components/runtime.rst

```diff
@@ -462,10 +458,23 @@ always using this ``ReactPHPRunner``::
 
 The end user will now be able to create front controller like::
 
-    require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
+    use App\Runtime\ReactPhpRuntime;
+    use Psr\Http\Server\RequestHandlerInterface;
 
-    return function (array $context): SomeCustomPsr15Application {
-        return new SomeCustomPsr15Application();
+    $_SERVER['APP_RUNTIME'] = ReactPhpRuntime::class;
+
+    require_once dirname(__DIR__) . '/vendor/autoload_runtime.php';
+
+    return static function (): RequestHandlerInterface {
+        return new class implements RequestHandlerInterface {
```

Comment:
I don't think the common use case for a react-php application would be to define your request handler as an anonymous class in the front controller, as this does really allow testing your logic (or at least it makes it harder). The previous code suggesting to instantiate a class makes more sense to me (however, changing the return type of the closure to `RequestHandlerInterface` might make sense).

Reply:
Hello @stof , sure, in fact this PR is to avoid frustration about testing this example with a minimal source code. Why not showing what "SomeCustomPsr15Application" could looks like ? (on another piece of code) It is for having a full functional example, for you is it too much ? In my mind, when I read it, I came to the fact it misses "things", maybe it is enough to stop at the "SomeCustomPsr15Application", but it requires more research if we want to enjoy the moment we end the reading to enjoy.